packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -210,10 +225,54 @@ export default class PackMoveOffersCtrl {
     set(offer, 'displayedPrice', displayedPrice);
   }
 
+  static updateSelectedGtrOption(value, offer, optionName) {
+    angular.forEach(offer.options, (option) => {
+      if (
+        option.name.match(/^gtr_/) &&
```

Comment:
**nitpick:** Some `String.prototype.match()` can be replaced by `String.prototype.startsWith()`.

Reply:
done